packages/components/src/public/QueryModel/withQueryModels.tsx

```diff
@@ -565,19 +650,26 @@ export function withQueryModels<Props>(
          * @param loadQueryInfo: boolean, if true will load the QueryInfo before loading the model's rows.
          * @param loadRows: boolean, if true will load the model's rows.
          * @param loadSelections: boolean, if true will load selections after loading QueryInfo.
+         * @param reloadTotalCount: boolean, if true will reload totalCount after loading QueryInfo.
          */
-        maybeLoad = (id: string, loadQueryInfo = false, loadRows = false, loadSelections = false): void => {
+        maybeLoad = (
+            id: string,
+            loadQueryInfo = false,
+            loadRows = false,
+            loadSelections = false,
+            reloadTotalCount = false
+        ): void => {
             if (loadQueryInfo) {
                 // Postpone loading any rows or selections if we're loading the QueryInfo.
                 this.loadQueryInfo(id, loadRows, loadSelections);
             } else {
-                // It's safe to load selections and rows in parallel.
-
                 if (loadRows) {
-                    this.loadRows(id);
-                }
+                    this.loadRows(id, loadSelections);
 
-                if (loadSelections) {
+                    if (this.state.queryModels[id].includeTotalCount) {
+                        this.loadTotalCount(id, reloadTotalCount);
+                    }
+                } else if (loadSelections) {
```

Comment:
Will this get called twice if both `loadRows` and `loadSelections` are true, or never get called with both true? If the latter is intended, perhaps we should log a warning or an error if they both are true (or have a different method for each, though that's probably more work than it's worth)?

Reply:
Yes, the `maybeLoad` will be called twice if both loadRows and loadSelections are true. The first time it will go into the if statement to call `this.loadRows` and then that will call `maybeLoad` again with loadRows: false and loadSelections: true, so the `this.loadSelections` will execute that time through.

Reply:
Got it. Thanks.